adr/0003-delay-schema-loading.md

```diff
@@ -0,0 +1,19 @@
+# 4. Delay schema loading
+
+Date: 2019-08-08
+
+## Status
+
+Accepted
+
+## Context
+
+The issue motivating this decision, and any context that influences or constrains the decision.
+
+## Decision
+
+The change that we're proposing or have agreed to implement.
+
+## Consequences
+
+What becomes easier or more difficult to do and any risks introduced by the change that will need to be mitigated.
```

Comment:
Basically the same as the AD above - schema could be loaded over the wire, we need to avoid it